values.md

```diff
@@ -1,4 +1,10 @@
-# Kubernetes Community Values
+---
+title: "Kubernetes Community Values"
+weight: 2
+description: |
+  These are the values that allow the community to function at its best.
```

Comment:
```suggestion
  The Kubernetes Community values are the keystone to the ongoing success of the project. These principles guide every aspect of the Kubernetes project.
```
This wording is what we use on the main k8s.io page: https://kubernetes.io/community/